.github/workflows/testing.yml

```diff
@@ -73,7 +73,9 @@ jobs:
           - type: tests
             goversion: '1.22'
             testflags: -race
-            grpcenv: 'GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST=true'
+            grpcenv: |
+              'GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST=true'
+              'GRPC_EXPERIMENTAL_ENABLE_PICK_FIRST_HAPPY_EYEBALLS=true'
```

Comment:
Optional: I feel like gating the new pick first policy with an environment variable is good enough. I don't feel the need for a second environment variable for this functionality, especially since both will eventually go default true. Or is the goal for a user to be able to just disable this algorithm even if using new pick first? I don't see many use cases for that, especially with the normal usage of endpoints. If that's the case, maybe a knob too (which I know the configs are hard defined in gRFC).

Reply:
PR description updated.